src/compiler/scala/tools/nsc/settings/Warnings.scala

```diff
@@ -97,6 +98,7 @@ trait Warnings {
   def warnInaccessible           = lint contains Inaccessible
   def warnNullaryOverride        = lint contains NullaryOverride
   def warnInferAny               = lint contains InferAny
+  def warnInferProductWithSerializable = lint contains InferProductWithSerializable
```

Comment:
I wouldn't be averse to using `warnInferPwS` everywhere. Then you preserve alignment, which is king.